src/richie/apps/core/cache.py

```diff
@@ -0,0 +1,140 @@
+"""
+Plugin for django-redis that supports redis sentinel.
+Credits :
+ - https://github.com/lamoda/django-sentinel
+ - https://github.com/KabbageInc/django-redis-sentinel
+"""
+
+import logging
+import random
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from django_redis.client import DefaultClient
+from redis.sentinel import Sentinel
+
+DJANGO_REDIS_LOGGER = getattr(settings, "DJANGO_REDIS_LOGGER", False)
```

Comment:
instead of `DJANGO_REDIS_LOGGER or __name__` below, why not default to `__name__` here?

But not sure why we would want to rename the logger and instantiate it in the class rather than here? This is not a pattern we use elsewhere.

Reply:
First, I agree with you, this is not a pattern we use elsewhere in our projects.
And this is not a choice of my own. But :

Since the `SentinelClient` is a `django-redis` add-on, it uses the same pattern than this library for logging in order to be consistent.

According to [django-redis documentation](https://github.com/jazzband/django-redis/blob/4a226ff18a9f0f8401104526fc54ce14de7f210f/README.rst#log-ignored-exceptions) :

> If you wish to specify the logger in which the exceptions are output, simply set the global variable DJANGO_REDIS_LOGGER to the string name and/or path of the desired logger. This will default to __name__ if no logger is specified and DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS is True:
> 
> DJANGO_REDIS_LOGGER = 'some.specified.logger'

And it is used like this [in the djange-redis library](https://github.com/jazzband/django-redis/blob/b7d64f0a80180ecfe8900407fbdf5e4196fa7a2a/django_redis/cache.py#L59-L63)...


Since this does not break anything and because there is a sane default value, I suggest to keep it like this.

Reply:
> This will default to name if no logger is specified

So maybe you don't need to worry about it since "name" is what we use on our side?

Reply:
Sorry, I misinterpreted a part of your comment !
I set `__name__` as default value here instead of `False`.